packages/instant/src/redux/async_data.ts

```diff
@@ -36,10 +36,11 @@ export const asyncData = {
             store.dispatch(actions.setAvailableAssets([]));
         }
     },
-    fetchAccountInfoAndDispatchToStore: async (store: Store) => {
+    fetchAccountInfoAndDispatchToStore: async (options: { store: Store; setLoading: boolean }) => {
```

Comment:
I changed to this syntax because for this and other methods, sending in booleans w/o this "psuedo-named parameter" approach felt indirect -- I found myself always looking up the function that was being invoked to understand what the boolean parameter represented.  With this approach, you can tell by the key of the object what the caller's intention is